### ISSUE TITLE
Supporting Gitlab Projects

### ISSUE THREAD

**Issue by safwanrahman** [peterbe/whatsdeployed#34]
Thanks a lot @peterbe for the application. Its very much useful for projects like Kuma, Kitsune and others.
I was thinking if you have any plan to support Gitlab hosted projects also? Currently it supports only github hosted projects.
If you agree to support Gitlab, I hope to spend some time on implementing it.

**Comment by peterbe** [comment]
Absolutely! It should be possible to have a checkbox that is "GitHub" by default or something. Then, whenever, it does what it does with GitHub it replaces that with GitLab. Go for it!
